resources/js/Components/LessonCard.jsx

```diff
@@ -1,31 +1,20 @@
-const LessonCard = ({ wordInJapanese, romanizedWord }) => {
+import React from "react";
+
+const LessonCard = (props) => {
```

Comment:
use destructuring here @brixtuyor so it's more cleaner

Reply:
Oh I forgot to edit it back, I was working on something when I did that. Will destructure it now...

Reply:
Done